src/Microsoft.IdentityModel.Tokens/AsymmetricAdapter.cs

```diff
@@ -395,244 +340,6 @@ private bool VerifyWithRsaCryptoServiceProviderProxyWithLength(byte[] bytes, int
-                            _rsaSignaturePaddingType,
-                            typeof(byte[]));
-
-                Delegate openDelegate = Delegate.CreateDelegate(delegateType, signDataMethod);
-                _rsaPkcs1SignMethod = (rsaArg, dataArg, algorithmArg) =>
-                {
-                    object[] args =
-                    {
-                        rsaArg,
-                        dataArg,
-                        Activator.CreateInstance(_hashAlgorithmNameType, algorithmArg),
-                        _rsaSignaturePaddingType.GetProperty("Pkcs1", BindingFlags.Static | BindingFlags.Public).GetValue(null)
-                    };
-
-                    return (byte[])openDelegate.DynamicInvoke(args);
-                };
-            }
-
-            return _rsaPkcs1SignMethod(RSA, input, _lightUpHashAlgorithmName);
-        }
-
-        private bool Pkcs1VerifyData(byte[] input, byte[] signature)
-        {
-            if (_rsaPkcs1VerifyMethod == null)
-            {
-                // [X] VerifyData(byte[] data, byte[] signature, HashAlgorithmName hashAlgorithmName, RSASignaturePadding padding)
-                // [ ] VerifyData(byte[] data, int offset, int count, byte[] signature, HashAlgorithmName hashAlgorithmName, RSASignaturePadding padding)
-                // [ ] VerifyData(Stream data, byte[] signature, HashAlgorithmName hashAlgorithmName, RSASignaturePadding padding)
-                Type[] signatureTypes = { typeof(byte[]), typeof(byte[]), _hashAlgorithmNameType, _rsaSignaturePaddingType };
-                MethodInfo verifyDataMethod = typeof(RSA).GetMethod(
-                    "VerifyData",
-                    BindingFlags.Public | BindingFlags.Instance,
-                    null,
-                    signatureTypes,
-                    null);
-
-                Type delegateType = typeof(Func<,,,,,>).MakeGenericType(
-                    typeof(RSA),
-                    typeof(byte[]),
-                    typeof(byte[]),
-                    _hashAlgorithmNameType,
-                    _rsaSignaturePaddingType,
-                    typeof(bool));
-
-                Delegate verifyDelegate = Delegate.CreateDelegate(delegateType, verifyDataMethod);
-                _rsaPkcs1VerifyMethod = (rsaArg, dataArg, signatureArg, algorithmArg) =>
-                {
-                    object[] args =
-                    {
-                        rsaArg,
-                        dataArg,
-                        signatureArg,
-                        Activator.CreateInstance(_hashAlgorithmNameType, algorithmArg),
-                        _rsaSignaturePaddingType.GetProperty("Pkcs1", BindingFlags.Static | BindingFlags.Public).GetValue(null)
-                    };
-
-                    return (bool)verifyDelegate.DynamicInvoke(args);
-                };
-            }
-
-            return _rsaPkcs1VerifyMethod(RSA, input, signature, _lightUpHashAlgorithmName);
-        }
-#endif
 #endregion
```

Comment:
ah this one needs to be removed (it matches with 
"#region NET45 'lightup' code")